.github/workflows/ci.yml

```diff
@@ -0,0 +1,120 @@
+name: Alfresco Transform Core CI
+
+on:
+  push:
+    branches:
+      - feature/**
+      - fix/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  pull_request:
+    branches:
+      - master
+      - feature/**
+      - fix/**
+      - SP/**
+      - HF/**
+      - ATS-**
+      - ACS-**
+      - MNT-**
+  workflow_call:
+  workflow_dispatch:
+
+env:
+  DOCKERHUB_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
+  DOCKERHUB_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  GITHUB_ACTIONS_DEPLOY_TIMEOUT: 60
```

Comment:
Let's move this line to the bottom compared to the other env vars, it makes it look more tidy.